openlibrary/components/BulkSearch/components/BulkSearchControls.vue

```diff
@@ -0,0 +1,152 @@
+
+
+
+<template>
+    <details open>
+        <summary>Input</summary>
+        <div>
+            <textarea v-model="bulkSearchState.inputText"></textarea>
+            <br />
+            <label>Format: <select @change="selectAlgorithm">
+                    <option value="1">e.g. "The Wizard of Oz by L. Frank Baum"</option>
+                    <option value="2">e.g. "The Wizard of Oz (L. Frank Baum)"</option>
+                    <option value="3">Wikipedia Citation (e.g. Baum, Frank L. (1994). The Wizard of Oz)</option>
+                    <option value="4">GPT</option>
+                </select></label>
+            <label v-if="bulkSearchState.extractionOptions.use_gpt">API-Key:
+                <input type="text" v-model="bulkSearchState.extractionOptions.api_key" />
+            </label>
+            <SampleBar @sample="(msg) => bulkSearchState.inputText = msg" />
+            <label>
+                <input checked v-model="bulkSearchState.matchOptions.includeAuthor" type="checkbox" /> Use author in
```

Comment:
Shouldn't specify this since it's defined by includeAuthor
```suggestion
                <input v-model="bulkSearchState.matchOptions.includeAuthor" type="checkbox" /> Use author in
```

Reply:
Removed!